### DISCUSSION TITLE
Microsteping and interpolation

### DISCUSSION THREAD

**Discussion by Kyrk98** [bassamanator/Sovol-SV06-firmware#190] [General]
In the printer.cfg file @bassamanator has used 128 micro steps and no interpolation ( to 256), this is a different approach to what Sovol and other manufacturer do to their default files where 16 microsteps and interpolation is used.  Granted, he doesn't use stelthchop but spreadcycle so the effects of interpolations don't matter that much, but then why use 128 microsteps and loose torque to achieve less noice? Have you find that this balance works better than the default 16 steps with interpolation and stealthchop overall?

**Comment by bassamanator** [comment]
Please read:
- Preface on this repo: https://github.com/bassamanator/Sovol-SV06-firmware?tab=readme-ov-file#preface
- https://github.com/bassamanator/Sovol-SV06-firmware/discussions/165#discussioncomment-9217084
- https://github.com/bassamanator/Sovol-SV06-firmware/discussions/34
- https://github.com/bassamanator/Sovol-SV06-firmware/discussions/28#discussioncomment-5598131
